modules/standard/JSON.chpl

```diff
@@ -80,13 +111,65 @@ module JSON {
       }
     }
 
+    /*
+      Start serializing a class by writing the character ``{``.
+
+      :arg writer: The ``fileWriter`` to be used when serializing.
+      :arg name: The name of the class type.
+      :arg size: The number of fields in the class.
+
+      :returns: A new :type:`AggregateSerializer`
+    */
+    proc startClass(writer: jsonWriter, name: string, size: int) throws {
+      writer.writeLiteral("{");
+      return new AggregateSerializer(writer, _ending="}");
+    }
+
+    /*
+      Start serializing a record by writing the character ``{``.
+
+      :arg writer: The ``fileWriter`` to be used when serializing.
+      :arg name: The name of the class type.
+      :arg size: The number of fields in the class.
+
+      :returns: A new :type:`AggregateSerializer`
+    */
+    proc startRecord(writer: jsonWriter, name: string, size: int) throws {
+      writer.writeLiteral("{");
+      return new AggregateSerializer(writer, _ending="}");
+    }
+
+    /*
+      Returned by ``startClass`` or ``startRecord`` to provide the API for
+      serializing classes or records.
+
+      Classes and records are serialized as JSON objects. For example, a
+      ``class`` or ``record`` with integer fields 'x' and 'y' with values '0'
```

Comment:
Should this be ``x``... ``0`` to be consistent with formatting for other examples?